src/main/java/appeng/me/helpers/AENetworkProxy.java

```diff
@@ -64,7 +64,7 @@
     private final String nbtName; // name
     private AEColor myColor = AEColor.TRANSPARENT;
     private CompoundNBT data = null; // input
-    private ItemStack myRepInstance = ItemStack.EMPTY;
```

Comment:
This is a safeguard as we do not check it for null anywhere and some addons are reusing our internal classes, which can lead to NPEs.

Reply:
No, it is set in every constructor. This was an IntelliJ warning (that the value set here is never used since every declared constructor overwrites it)

Reply:
Ah yeah. in that case wrapping it into an `Objects.requireNonNull` would be better. And probably other fields like `gp`.